policies/environments/expected.rego

```diff
@@ -38,6 +38,7 @@ expected =
     "maatdb",
     "mlra",
     "mojfin",
+    "ncas",
```

Comment:
I don't think it needs to be ordered alphabetically, but since we've kept it that way so far, it's probably worth continuing